### ISSUE TITLE
Database Cleaning is slow

### ISSUE THREAD

**Issue by lewispb** [DEFRA/ruby-services-team#11]
Our current implementation of a database cleaner is extremely slow. Each test requires clearing Mongo, Redis and Elastic Search. The Elastic Search service in particular does not cope well with instant cleaning and indexing and so sometimes requires a `sleep` to prevent a test failure. The cleaning script also prevents us running the tests in parallel.

This really degrades the experience of running the cucumber test suite which takes almost 10 minutes to run.

We should investigate if there is a way of speeding this essential process up, and perhaps only cleaning elastic search on required tests rather than before every test.

**Comment by Cruikshanks** [comment]
WCR no longer uses Elasticsearch, plus it is our intention to migrate the remaining functionality in the frontend into our new apps.

Hence have moved this across in the spirit of having a single list of issues, but it is not our intention to fix it.

**Comment by irisfaraway** [comment]
Closing as we've agreed we won't fix this.
